build/nimiq-1.0/README.md

```diff
@@ -1,6 +1,6 @@
 ---
 search: false
 ---
-# Nimiq 1.0
+# Nimiq PoW
 
-This is a port of the old content of the Nimiq 1.0.
+This is a port of the old content of the Nimiq PoW.
```

Comment:
The Nimiq PoW _what_? There's a word missing at the end of the sentence. _Documentation_? _Build instructions_?

Reply:
It is just a README.md. It does not get shown in the website: https://www.nimiq.com/developers/build/nimiq-1.0/